lib/codit/population/person.py

```diff
@@ -66,7 +66,6 @@ def isolating(self):
 
```

Comment:
```python

@property
def self.immunities():
    """
    The idea is that the immunities a person have are a simple dictionary lookup of their covid_experiences
    """
    immunities = set()
    for d in self.covid_experiences:
        immunities.add(self.cfg.CROSS_IMMUNITY[d])
    # for v in self.vacciations:
    #    immunities.add(self.cfg.VACCINATION_IMMUNITY[v])
    return immunities

```

so that then we can write code talking about `self.immunities`